test/scss/_clearfix.scss

```diff
@@ -4,15 +4,15 @@
 
 // Clearfix
 // --------
-@include test-module('clearfix [mixin]') {
-  @include test('Clearfix it!') {
+@include describe('clearfix [mixin]') {
+  @include it('Clearfix it!') {
     @include assert('Adds a micro-clearfix to an element') {
```

Comment:
are we removing the reference of "micro-" everywhere?

Reply:
It is still a micro-clearfix, so this doesn't bother me.